### PR TITLE
Jobs improvements helpcenter documentation

### PR DESCRIPTION
**Pull request by felix-couprie**
- Stop job
- Loading bar
- Time estimation
- New UI badges
- New job status
- New ACL

### CODE REVIEW

File: content/md/imports-exports/monitor-jobs.md [hunk 1]
```diff
@@ -9,48 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
 1.  Click on your import/export
-1.  The page of the import/export job detail should appear
+1.  The import/export job details appear
 
 ![image](../img/Exports_Calculation5.png)
 
 The details of the executing import/export job are shown on the page. The details are related to each executed import/export profile.
 
-## Mass actions execution reports
+## Mass actions execution jobs
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:63]:
```suggestion
## Mass action execution jobs
```

File: content/md/imports-exports/monitor-jobs.md [hunk 2]
```diff
@@ -9,48 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
 
-You can also have more details when you executed a mass action. To review their report:
+You can have more details when you execute a mass action. To review their report:
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:65]:
```suggestion
You can have more details about executing a mass action. To check the report:
```

Comment (camillebourjade) [reply content/md/imports-exports/monitor-jobs.md:65]: report or job detail?

File: content/md/imports-exports/monitor-jobs.md [hunk 3]
```diff
@@ -9,48 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
 1.  Go to `Activity`/`Process Tracker` or click on the notification of the completed mass action
-1.  The page of the report is then displayed: the details of the executing bulk action are shown on the page
+1.  The page of the report is then displayed with the details of the executed mass action
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:67]:
```suggestion
1.  The page of the report is displayed, it contains all the details about the executed mass action.
```

File: content/md/imports-exports/monitor-jobs.md [hunk 4]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
 - By status
 
 ![Process tracker search filters](../img/Exports_Process-Tracker-search-filters.png)
+
+# How to stop your jobs?
+
+If you launched a job by error, or for any other reason, it is possible to stop an ongoing job. The jobs that can be stopped are the following:
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:83]:
```suggestion
It is possible to stop a running job if it was launched by mistake or if you need to review a product selection for instance. The jobs that can be stopped are the following:
```

File: content/md/imports-exports/monitor-jobs.md [hunk 5]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+* All export jobs
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:84]:
```suggestion
* Export jobs
```

File: content/md/imports-exports/monitor-jobs.md [hunk 6]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+* All import jobs (except **families** and **family variants**)
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:85]:
```suggestion
* Import jobs (except **families** and **family variants**)
```

File: content/md/imports-exports/monitor-jobs.md [hunk 7]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+* Mass actions (mass delete products, mass publish products, mass unpublish products)
+* Rules (rules execution, calculation of impacted products)
+* Proposals (approve all, reject all)
+
+Note that **stopping a job won't revert the actions already performed**: for instance, if you launched a mass action about an attribute for a selection of products and you realise a few minutes after that you selected the wrong products, if you stop the ongoing job, the modifications done before you stopped the job won't be reverted. You will have to mass edit back the modifications.
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:90]:
```suggestion
Note that **stopping a job won't revert the actions already performed**: for instance, if you launched a mass action about an attribute for a selection of products and you realize a few minutes later that you selected the wrong products, if you stop the ongoing job, the modifications done before you stopped the job won't be reverted. You will have to mass edit the modifications back.
```

File: content/md/imports-exports/monitor-jobs.md [hunk 8]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+
+To stop a job, there are different ways described in the sections below.
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:92]:
```suggestion
To stop a job, there are different ways that you will find in the sections below.
```

File: content/md/imports-exports/monitor-jobs.md [hunk 9]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+1.  Under `Last operations`, look for the ongoing job you would like to stop
+1.  Click on `STOP` at the end of the job's line
+
+## Stoppping a job from the Process Tracker
+
+1.  Go to `Activity` then `Process Tracker`
+1.  Search for the ongoing job you would like to stop
+1.  Click on `STOP` at the end of the job's line
+
+## Stoppping a job from the job details itself
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:110]:
```suggestion
## Stopping a job from the job detail itself
```

File: content/md/imports-exports/monitor-jobs.md [hunk 10]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+
+Note that **stopping a job won't revert the actions already performed**: for instance, if you launched a mass action about an attribute for a selection of products and you realise a few minutes after that you selected the wrong products, if you stop the ongoing job, the modifications done before you stopped the job won't be reverted. You will have to mass edit back the modifications.
+
+To stop a job, there are different ways described in the sections below.
+
+:::info
+The ability to stop a job is defined at the role-level. Please read our [Interface and actions accesses](manage-the-interface-and-actions-accesses.html#rights-on-system-pages) documentation for more information.
+:::
+
+## Stoppping a job from the Dashboard
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:98]:
```suggestion
## Stopping a job from the Dashboard
```

File: content/md/imports-exports/monitor-jobs.md [hunk 11]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+
+1.  Go to `Activity` then `Dashboard`
+1.  Under `Last operations`, look for the ongoing job you would like to stop
+1.  Click on `STOP` at the end of the job's line
+
+## Stoppping a job from the Process Tracker
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:104]:
```suggestion
## Stopping a job from the Process Tracker
```

File: content/md/imports-exports/monitor-jobs.md [hunk 12]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+1.  Click on `STOP` at the end of the job's line
+
+## Stoppping a job from the job details itself
+
+1.  Go to `Activity` then `Process Tracker`
+1.  Search for the ongoing job you would like to stop
+1.  Click on the desired job
+1.  In the top right corner, click on `STOP`
+
+## Stoppping a job from an export/import profile
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:117]:
```suggestion
## Stopping a job from an export/import profile
```

File: content/md/imports-exports/monitor-jobs.md [hunk 13]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+
+1.  Go to `Activity` then `Process Tracker`
+1.  Search for the ongoing job you would like to stop
+1.  Click on `STOP` at the end of the job's line
+
+## Stoppping a job from the job details itself
+
+1.  Go to `Activity` then `Process Tracker`
+1.  Search for the ongoing job you would like to stop
+1.  Click on the desired job
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:114]:
```suggestion
1.  Click on the job you'd like to stop
```

File: content/md/imports-exports/monitor-jobs.md [hunk 14]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+1.  In the top right corner, click on `STOP`
+
+## Stoppping a job from an export/import profile
+
+1.  Go to `Imports` or `Exports`
+1.  Click on the desired import or export profile
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:120]:
```suggestion
1.  Click on the right import or export profile
```

File: content/md/imports-exports/monitor-jobs.md [hunk 15]
```diff
@@ -63,3 +77,46 @@ If you want to find a specific job amongst all those listed in the process track
...
+1.  Under `Last execution`, look for the ongoing job you would like to stopped
```
Comment (camillebourjade) [inline content/md/imports-exports/monitor-jobs.md:121]:
```suggestion
1.  Under `Last execution`, look for the ongoing job you would like to stop
```

File: content/md/permissions-settings/manage-the-interface-and-actions-accesses.md [hunk 16]
```diff
@@ -282,3 +282,4 @@ Only give UI permissions to user roles that are related to real users, not to [c
 | Manage the API connections | User can access the page `System`/`API connections`  |
 | View process tracker | User can access the page `System`/`Process tracker` and view his own jobs |
```
Comment (camillebourjade) [inline content/md/permissions-settings/manage-the-interface-and-actions-accesses.md:283]:
```suggestion
| View process tracker | User can access the page `System`/`Process tracker` and view their own jobs |
```

File: content/md/imports-exports/monitor-jobs.md (hunk 2 above)
Comment (felix-couprie) [reply content/md/imports-exports/monitor-jobs.md:65]: "Job detail"

**Comment by felix-couprie** [comment]
> English fixes

All applied. Thanks :)

File: content/md/imports-exports/monitor-jobs.md [hunk 17]
```diff
@@ -9,50 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
-
-There is a Process Tracker page which will give you the full list of all jobs you launched. To access it, go to `Activity` and click on the `Process tracker` menu entry.
-![image](../img/Settings_Calculation3.png)
-
-The `Last operations` widget on Akeneo dashboard will also give you the status of your launched jobs.
-![image](../img/Dashboard_Calculation4.png)
+- a [mass actions](product-mass-actions.html),
+- a calculation of [project](what-is-a-project.html)
+- a [rule](what-is-a-rule.html) execution _(EE only)_
+- the approval/rejection of [approvals](proposals-workflow.html) _(EE only)_
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:18]:
```suggestion
- the approval/rejection of [proposals](proposals-workflow.html) _(EE only)_
```

File: content/md/imports-exports/monitor-jobs.md [hunk 18]
```diff
@@ -9,50 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
-
-Then, when the job is completed, a notification appears on the top of your screen.
-![image](../img/Settings_Calculation2.png)
-
-There is a Process Tracker page which will give you the full list of all jobs you launched. To access it, go to `Activity` and click on the `Process tracker` menu entry.
-![image](../img/Settings_Calculation3.png)
-
-The `Last operations` widget on Akeneo dashboard will also give you the status of your launched jobs.
-![image](../img/Dashboard_Calculation4.png)
+- a [mass actions](product-mass-actions.html),
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:15]:
```suggestion
- a [mass action](product-mass-actions.html),
```

File: content/md/imports-exports/monitor-jobs.md [hunk 19]
```diff
@@ -9,50 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
+- a calculation of [project](what-is-a-project.html)
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:16]:
```suggestion
- a calculation of [project](what-is-a-project.html) _(EE only)_
```

File: content/md/imports-exports/monitor-jobs.md [hunk 20]
```diff
@@ -9,50 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
+1.  The import/export job detail appear
+![Jobs export/import profiles](../img/Jobs_export-import-profiles.png)
 
-## Mass actions execution reports
+The details of the executed import/export job are displayed and related to individual import/export profiles.
 
-You can also have more details when you executed a mass action. To review their report:
-1.  Go to `Activity`/`Process Tracker` or click on the notification of the completed mass action
-1.  The page of the report is then displayed: the details of the executing bulk action are shown on the page
+## Mass actions jobs
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:61]: What about other jobs such as a rule execution or a project calculation?
You can be less specific in the title to cover them as well.

File: content/md/imports-exports/monitor-jobs.md [hunk 21]
```diff
@@ -9,50 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
-The details of the executing import/export job are shown on the page. The details are related to each executed import/export profile.
+You can get more details when you execute an import or an export job. To access the job detail:
+1.  Go to `Imports` or `Exports`
+1.  In the list, select the profile job you need
+1.  Click on your import/export job
+1.  The import/export job detail appear
+![Jobs export/import profiles](../img/Jobs_export-import-profiles.png)
 
-## Mass actions execution reports
+The details of the executed import/export job are displayed and related to individual import/export profiles.
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:59]: As a info note, you can also add that those jobs are also visible in the Process tracker page.

File: content/md/imports-exports/monitor-jobs.md [hunk 22]
```diff
@@ -61,5 +73,44 @@ If you want to find a specific job amongst all those listed in the process track
...
+# How to stop your jobs?
+
+It is possible to stop a running job if it was launched by mistake or if you need to review a product selection for instance. The jobs that can be stopped are the following:
+* Export jobs
+* Import jobs (except **families** and **family variants**)
+* Mass actions (mass delete products, mass publish products, mass unpublish products)
+* Rules (rules execution, calculation of impacted products)
+* Proposals (approve all, reject all)
+
+Note that **stopping a job won't revert the actions already performed**: for instance, if you launched a mass action about an attribute for a selection of products and you realize a few minutes later that you selected the wrong products, if you stop the ongoing job, the modifications done before you stopped the job won't be reverted. You will have to mass edit the modifications back.
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:87]: I'd suggest adding a warning as it is pretty important. ;) 
```suggestion
:::warning
Note that **stopping a job won't revert the actions already performed**: for instance, if you launched a mass action about an attribute for a selection of products and you realize a few minutes later that you selected the wrong products, if you stop the ongoing job, the modifications done before you stopped the job won't be reverted. You will have to mass edit the modifications back.
:::
```

File: content/md/imports-exports/monitor-jobs.md [hunk 23]
```diff
@@ -61,5 +73,44 @@ If you want to find a specific job amongst all those listed in the process track
...
+
+:::info
+The ability to stop a job is defined at the role-level. Please read our [Interface and actions accesses](manage-the-interface-and-actions-accesses.html#rights-on-system-pages) documentation for more information.
+:::
+
+## Stopping a job from the Process Tracker
+
+1.  Go to `Activity` then `Process Tracker`
+1.  Search for the ongoing job you would like to stop
+1.  Click on `STOP` at the end of the job's line
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:99]:
```suggestion
1.  Click on `Stop` at the end of the job's line
```

File: content/md/imports-exports/monitor-jobs.md [hunk 24]
```diff
@@ -61,5 +73,44 @@ If you want to find a specific job amongst all those listed in the process track
...
+![Stopping a job from the process tracker](../img/Jobs_stop-job-process-tracker.png)
+
+## Stopping a job from the job detail itself
+
+1.  Go to `Activity` then `Process Tracker`
+1.  Search for the ongoing job you would like to stop
+1.  Click on the job you'd like to stop
+1.  In the top right corner, click on `STOP`
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:107]:
```suggestion
1.  In the top right corner, click on `Stop`
```

File: content/md/imports-exports/monitor-jobs.md [hunk 25]
```diff
@@ -61,5 +73,44 @@ If you want to find a specific job amongst all those listed in the process track
...
+![Stopping a job from the job detail](../img/Jobs_stop-job-detail.png)
+
+## Stopping a job from an export/import profile
+
+1.  Go to `Imports` or `Exports`
+1.  Click on the right import or export profile
+1.  Under `Last execution`, look for the ongoing job you would like to stop
+1.  Click on `STOP` at the end of the job's line
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:115]:
```suggestion
1.  Click on `Stop` at the end of the job's line
```

File: content/md/imports-exports/monitor-jobs.md [hunk 26]
```diff
@@ -61,5 +73,44 @@ If you want to find a specific job amongst all those listed in the process track
...
+
+Note that **stopping a job won't revert the actions already performed**: for instance, if you launched a mass action about an attribute for a selection of products and you realize a few minutes later that you selected the wrong products, if you stop the ongoing job, the modifications done before you stopped the job won't be reverted. You will have to mass edit the modifications back.
+
+To stop a job, there are different ways that you will find in the sections below.
+
+:::info
+The ability to stop a job is defined at the role-level. Please read our [Interface and actions accesses](manage-the-interface-and-actions-accesses.html#rights-on-system-pages) documentation for more information.
+:::
+
+## Stopping a job from the Process Tracker
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:95]:
```suggestion
## Stop a job from the Process Tracker
```

File: content/md/imports-exports/monitor-jobs.md [hunk 27]
```diff
@@ -61,5 +73,44 @@ If you want to find a specific job amongst all those listed in the process track
...
+
+1.  Go to `Activity` then `Process Tracker`
+1.  Search for the ongoing job you would like to stop
+1.  Click on `STOP` at the end of the job's line
+![Stopping a job from the process tracker](../img/Jobs_stop-job-process-tracker.png)
+
+## Stopping a job from the job detail itself
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:102]:
```suggestion
## Stop a job from the job detail itself
```

File: content/md/imports-exports/monitor-jobs.md [hunk 28]
```diff
@@ -61,5 +73,44 @@ If you want to find a specific job amongst all those listed in the process track
...
+
+1.  Go to `Activity` then `Process Tracker`
+1.  Search for the ongoing job you would like to stop
+1.  Click on the job you'd like to stop
+1.  In the top right corner, click on `STOP`
+![Stopping a job from the job detail](../img/Jobs_stop-job-detail.png)
+
+## Stopping a job from an export/import profile
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:110]:
```suggestion
## Stop a job from an export/import profile
```

File: content/md/imports-exports/monitor-jobs.md [hunk 29]
```diff
@@ -9,50 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
-1.  Go to `Activity`/`Process Tracker` or click on the notification of the completed mass action
-1.  The page of the report is then displayed: the details of the executing bulk action are shown on the page
+## Mass actions jobs
 
-![image](../img/Dashboard_Calculation6.png)
+You can have more details about executing a mass action. To check the report:
+1.  Go to `Activity`/`Process Tracker`
+1.  Search for the job you would like to know more about
+1.  Click on the job to get more details
+![Jobs mass execution details](../img/Jobs_mass-actions.png)
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:67]: The image has weird grey areas at the top and bottom. Maybe coming from the Figma filter?

File: content/md/imports-exports/monitor-jobs.md [hunk 30]
```diff
@@ -9,57 +9,91 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
+1.  First, you are notified by a flash message when a job starts. Friendly warning, the approval/rejection of proposals don't trigger flash messages.
+![Jobs flash message](../img/Jobs_flash-message.png)
+1. Then, while the job is running, it will be listed in the Process Tracker amongst all the other jobs you launched. To access it, go to `Activity` and click on `Process tracker`.
+![List of jobs in process tracker](../img/Jobs_process-tracker.png)  
+Note that you can also check the status of your launched jobs on the dashboard. To access it, go to `Activity` and click on `Dashboard`.
+![List of jobs in dashboard](../img/Jobs_dashboard.png)
+1.  Finally, when the job is complete, a notification appears at the top of your screen. You can always go back to the job itself if you need more details.
+![Notification of a job complete](../img/Jobs_notification.png)
 
-# How to monitor?
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:19]: As you removed this title, can you make sure it's not referenced somewhere else in the help center? If it's the case, you'll need to change the link to another one in order to make it work again.

File: content/md/imports-exports/monitor-jobs.md [hunk 31]
```diff
@@ -9,57 +9,91 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
...
+If you want to find a specific job amongst all those listed in the process tracker, you can use the following search filters:
+* By job name
+* By type
+* By user
+* By status
+![Process tracker search filters](../img/Exports_Process-Tracker-search-filters.png)
+
+# How to stop your jobs?
 
-## Imports & exports reports
```
Comment (NolwennP) [inline content/md/imports-exports/monitor-jobs.md:36]: As you removed this title, can you make sure it's not referenced somewhere else in the help center? If it's the case, you'll need to change the link to another one in order to make it work again.

File: content/md/imports-exports/monitor-jobs.md (hunk 20 above)
Comment (felix-couprie) [reply content/md/imports-exports/monitor-jobs.md]: I re-ordered the paragraphs, thanks

File: content/md/imports-exports/monitor-jobs.md (hunk 30 above)
Comment (felix-couprie) [reply content/md/imports-exports/monitor-jobs.md:19]: Done, 1 correction made

File: content/md/imports-exports/monitor-jobs.md (hunk 31 above)
Comment (felix-couprie) [reply content/md/imports-exports/monitor-jobs.md:36]: Checked, nothing to modify here. Thanks

**Review by NolwennP — APPROVED** [review]

**Comment by felix-couprie** [comment]
Thanks a lot @NolwennP

**Pull request merged**